target/linux/ipq40xx/files/arch/arm/boot/dts/qcom-ipq4019-cm520-79f.dts

```diff
@@ -197,13 +201,11 @@
 };
 
 &gmac0 {
-	nvmem-cells = <&macaddr_art_1006>;
-	nvmem-cell-names = "mac-address";
+	mtd-mac-address = <&art 0x1006>;
```

Comment:
NACK, unless there is a really good reason, the same applies for NVMEM getting dropped in other nodes